test-galasactl-ecosystem.sh

```diff
@@ -178,6 +185,143 @@ function launch_test_on_ecosystem_with_portfolio {
+function runs_get_check_summary_format_output {
+    h2 "Performing runs get with summary format..."
+
+    run_name=$1
+
+    cd ${BASEDIR}/temp
+
+    cmd="${BASEDIR}/bin/${binary} runs get \
+    --name ${run_name} \
+    --format summary \
+    --bootstrap ${bootstrap} "
+
+    info "Command is: $cmd"
+
+    output_file="runs-get-output.txt"
+    $cmd | tee $output_file
+
+    # Check that the full test name is output
+    cat $output_file | grep "${GALASA_TEST_NAME_LONG}"
+    rc=$?
+    # We expect a return code of '0' because the test name should be output.
+    if [[ "${rc}" != "0" ]]; then 
+        error "Did not find ${GALASA_TEST_NAME_LONG} in summary output"
+        exit 1
+    fi
+
+    # Check headers
+    headers=("name" "status" "result" "test-name")
+
+    for header in "${headers[@]}"
+    do
+        cat $output_file | grep "$header"
+        rc=$?
+        # We expect a return code of '0' because the header name should be output.
+        if [[ "${rc}" != "0" ]]; then 
+            error "Did not find header $header in summary output"
+            exit 1
+        fi
+    done    
+
+    # Check that we got 2 lines out... one for the headers, on for the 1 line of test data.
+    line_count=$(cat $output_file | wc -l | xargs)
+    expected_line_count="2"
+    if [[ "${line_count}" != "${expected_line_count}" ]]; then 
+        error "line count is wrong. expected ${expected_line_count} got ${line_count}"
+        exit 1
+    fi
+
+    success "galasactl runs get --format summary seemed to work"
+}
+
+#--------------------------------------------------------------------------
+function runs_get_check_details_format_output {
+    h2 "Performing runs get with details format..."
+
+    run_name=$1
+
+    cd ${BASEDIR}/temp
+
+    cmd="${BASEDIR}/bin/${binary} runs get \
+    --name ${run_name} \
+    --format details \
+    --bootstrap ${bootstrap} "
+
+    info "Command is: $cmd"
+
+    output_file="runs-get-output.txt"
+    $cmd | tee $output_file
+
+
+    # Check that the full test name is output and formatted
+    cat $output_file | grep "test-name    :  ${GALASA_TEST_NAME_LONG}"
+    rc=$?
+    # We expect a return code of '0' because the ecosystem should be able to find this test as we just ran it.
+    if [[ "${rc}" != "0" ]]; then 
+        error ""
+        exit 1
+    fi
+
+    # Check method headers
+    headers=("method" "type" "status" "result" "start-time" "end-time" "duration(ms)")
+
+    for header in "${headers[@]}"
+    do
+        cat $output_file | grep "$header" $output_file
+        rc=$?
+        # We expect a return code of '0' because the header name should be output.
+        if [[ "${rc}" != "0" ]]; then 
+            error "Did not find header $header in details output"
+            exit 1
+        fi
+    done  
+
+    #check methods start on line 13 - implies other test details have outputted 
+    line_count=$(grep -n "method" $output_file | head -n1 | sed 's/:.*//')
+    expected_line_count="13"
```

Comment:
Can the 13 be some constant defined near the testcase name ?
eg: GALASA_TEST_RUN_GET_EXPECTED_DETAILS_LINE_COUNT="13"